setup.py

```diff
@@ -49,8 +49,6 @@ def read(fname):
 extras_require = {
     'docs': ['sphinx'],
     'tests': test_requires,
-    ': platform_python_implementation != "PyPy"': ['psycopg2'],
```

Comment:
Some users need to install vanilla `pscyopg2` because they don't trust pre-compiled wheels or there isn't one for their platform, so I think it should be left up to the user.